### PR TITLE
Admin page

### PR DESCRIPTION
**Pull request by SetterVet**

### CODE REVIEW

**Comment by SetterVet** [comment]
ready to review @afurm @VasylYaremkiv @rubycop @sonishc

File: .rubocop.yml [hunk 1]
```diff
@@ -21,3 +21,5 @@ Metrics/BlockLength:
   Exclude:
     - "**/*_spec.rb"
     - "db/schema.rb"
+Metrics/CyclomaticComplexity:
+  Max: 10
```
Comment (VasylYaremkiv) [inline .rubocop.yml]: Why do you set 10 ?

File: app/policies/application_policy.rb [hunk 2]
```diff
@@ -5,4 +5,29 @@ def initialize(user, record)
     @user = user
     @record = record
   end
+
+  def rails_admin?(action)
+    case action
```
Comment (VasylYaremkiv) [inline app/policies/application_policy.rb]: Method is too long. Use `include?` and move to constant

File: .rubocop.yml (hunk 1 above)
Comment (SetterVet) [reply .rubocop.yml]: case in ApplicationPolicy
@VasylYaremkiv

Comment (VasylYaremkiv) [reply .rubocop.yml]: You should fix rubocop warning and do not change rubocop config

File: app/policies/application_policy.rb [hunk 3]
```diff
@@ -5,4 +5,12 @@ def initialize(user, record)
     @user = user
     @record = record
   end
+
+  def rails_admin?(action)
+    rails_admin_action = %i[dashboard index show new edit destroy export history show_in_app]
```
Comment (rubycop) [inline app/policies/application_policy.rb]: Lets move it to constant

**Comment by SetterVet** [comment]
fixed @VasylYaremkiv @afurm @rubycop @sonishc

**Comment by SetterVet** [comment]
make constant @rubycop @VasylYaremkiv @afurm @sonishc

**Pull request merged**